documentation/_templates/index.html

```diff
@@ -111,6 +111,8 @@ <h2>Documentation</h2>
           <span class="linkdescr">significant contributors</span></p>
        <p class="biglink"><a class="biglink" href="{{ pathto("documentation/PUBLICATIONS") }}">Publications</a><br/>
           <span class="linkdescr">papers and talks about (or using) FiPy</span></p>
+       <p class="biglink"><a class="biglink" href="{{ pathto("CHANGELOG") }}">Change Log</a><br/>
```

Comment:
Red highlighting suggests these quotes may conflict. Is it possible to write `href="{{ pathto('documentation/PUBLICATIONS') }}"` and `href="{{ pathto('CHANGELOG') }}"` instead? Or is GitHub being overly cautious?

Reply:
That whole section has been like that forever and not caused problems, but I agree that it's goofy. I need to research the templating language to see what characters are allowed.

Reply:
Confirmed that Jinja allows for single quotes and made the change